packages/mpmath/patches/fix-warnings.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/mpmath/ctx_mp_python.py b/mpmath/ctx_mp_python.py
```

Comment:
Could you please add a comment about the fact that this should be removed with the next mpmath release? Otherwise LGTM.